_source/_includes/setup/cli.md

```diff
@@ -27,7 +33,7 @@ Select **
     {%- else -%}Other
     {%- endif -%}**. 
     {% if include.loginRedirectUri and include.logoutRedirectUri %}Then, change the Redirect URI to `{{ include.loginRedirectUri }}` and use `{{ include.logoutRedirectUri }}` for the Logout Redirect URI.
-    {% else %}Accept the default Redirect URI values provided for you.
+    {% else %}Accept the default Redirect URI values provided for you.{% if include.framework contains "Spring Boot" %} That is, a Login Redirect of `\http://localhost:8080/login/oauth2/code/okta` and a Logout Redirect of `\http://localhost:8080`.{% endif %}
```

Comment:
With Asciidoctor, the backslashes prevent the URL from being a link. With Markdown, the backslash shows up, so this will need a conditional.